src/styles/blocks/header.scss

```diff
@@ -0,0 +1,22 @@
+.header {
+  box-sizing: border-box;
+  height: 100vh;
+  background-image: url(../images/header-bg-mobile.png);
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__title {
+    font-family: "Space Grotesk", sans-serif;
```

Comment:
Consider using a variable for the font family, as it makes it easier to maintain and keep the code consistent.